src/components/Item.jsx

```diff
@@ -17,6 +32,26 @@ const Item = ({ name, id, lastPurchasedDate }) => {
       return now - lastPurchasedDate.seconds < day;
     }
   };
+
+  const getPurchaseInterval = () => {
+    let lastEstimate = frequency;
+    let latestInterval =
```

Comment:
I spent a little bit of time understanding why we need to calculate this `lastInterval` value to pass into the `calculateEstimate` function and it became a little clearer to me that it's representing the number of days since the last purchase.

I think if I were to have written `calculateEstimate` I would have just accepted the last purchase date instead and handled this calculation there.

The way it's written now satisfies the function's api. 👍

The only feedback I have for this calculation is how we're doing the date calculation by converting to a number. It may cause some weirdness for people who are sharing a list across timezones. Ideally, we'd want to operate on the date object itself. I don't currently have a good example of this though.

Reply:
Thanks for this Kevin, we've decided to leave the date calculation as is due to time constraints, but definitely  worth looking into for future projects 👍🏾

Reply:
That will work fine for now, just keep it in mind when working on future projects. 👍